src/main/java/io/phasetwo/service/model/jpa/OrganizationAdapter.java

```diff
@@ -165,15 +165,23 @@ public void setAttribute(String name, List<String> values) {
   @Override
   public Stream<UserModel> searchForMembersStream(
       String search, Integer firstResult, Integer maxResults) {
+    String[] searchTerms = Strings.isNullOrEmpty(search) ? null : search.split(",");
```

Comment:
This should be an empty array, rather than null, because the `for` below will throw an error if it's null.